airflow-core/src/airflow/ui/public/i18n/locales/de/components.json

```
@@ -88,6 +88,11 @@
     "taskGroup": "Task Gruppe"
   },
   "limitedList": "+{{count}} mehr",
+  "limitedList.allItems": "Alle {{count}} Einträge:",
+  "limitedList.clickToInteract": "Klicken Sie auf ein Tag, um DAGs zu filtern",
```

```suggestion
  "limitedList.clickToInteract": "Klicken Sie auf ein Tag, um Dags zu filtern",
```

---

In other areas we had translated "Tag" with "Markierungen". So the proposal is (unfortunately) inconsistent.
```suggestion
  "limitedList.clickToInteract": "Klicken Sie auf eine Markierung, um Dags zu filtern",
```